mergify_engine/web/__init__.py

```diff
@@ -46,20 +46,26 @@
 app.mount("/badges", badges.app)
 
 
+_AREDIS_STREAM: aredis.StrictRedis = None
+_AREDIS_CACHE: aredis.StrictRedis = None
```

Comment:
sniff...

Reply:
one could argue it's slightly cleaner that attaching things to an object like a cow-boy

Reply:
That's fastapi recommendation !